tools/integration/src/index.ts

```diff
@@ -758,126 +774,318 @@ async function handleExport(argv: any) {
-        const dashboard = await exportDashboard(dashboardId);
-        saveDashboard(dashboardsPath, dashboardId, dashboard.title, dashboard)
-    } else {
-        const idObjects = await getDashboardList();
+    return result;
+}
+
+async function exportEvent(server: string, token: string, eventId: string, axiosInstance: any): Promise<any> {
+    try {
+        const url = `https://${server}/api/events/settings/event-specifications/custom/${eventId}`;
+        logger.info(`Getting event (id=${eventId}) from ${url}`);
 
-        const filteredIdObjects = filterDashboardsBy(idObjects, includeConditions);
-        const sanitizedIdObjects = sanitizeDashboardTitles(filteredIdObjects)
+        const response = await axiosInstance.get(url, {
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': `apiToken ${token}`
+            }
+        });
 
-        for (const obj of sanitizedIdObjects) {
-            const dashboard = await exportDashboard(obj.id);
-            saveDashboard(dashboardsPath, obj.id, obj.title, dashboard)
+        logger.info(`Successfully got event (id=${eventId}): ${response.status}`);
+        if (logger.isDebugEnabled()) {
+            logger.debug(`Response data: \n${JSON.stringify(response.data)}`);
         }
 
-        logger.info(`Total dashboard(s) processed: ${sanitizedIdObjects.length}`)
+        return response.data;
+    } catch (error) {
+        handleAxiosError(error, `event (id=${eventId})`);
+        return null;
     }
 }
 
-function printDirectoryTree(dirPath: string, rootLabel: string, indent: string = ''): void {
-    const isRoot = indent === '';
-    if (isRoot) {
-      logger.info(rootLabel);
+async function getEventList(server: string, token: string, axiosInstance: any): Promise<any[]> {
+    try {
+        const url = `https://${server}/api/events/settings/event-specifications/custom`;
+        logger.info(`Getting event list from ${url}`);
+
+        const response = await axiosInstance.get(url, {
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': `apiToken ${token}`
+            }
+        });
+
+        logger.info(`Successfully got event list: ${response.status}`);
+        if (logger.isDebugEnabled()) {
+            logger.debug(`Response data: \n${JSON.stringify(response.data)}`);
+        }
+
+        return response.data;
+    } catch (error) {
+        handleAxiosError(error, `event list`);
+        return [];
     }
+}
 
-    const files = fs.readdirSync(dirPath);
-    const lastIndex = files.length - 1;
+function saveEvent(dir: string, id: string, name: string, event: any) {
+    try {
+        const filename = `${name}.json`;
+        const filepath = path.join(dir, filename);
+        logger.info(`Saving event (id=${id}) to ${filepath}`);
+        fs.writeFileSync(filepath, JSON.stringify(event, null, 2));
+        logger.info(`Event (id=${id}) saved successfully`);
```

Comment:
Please be sure that we use the term consistently w/o ambiguity when it comes to the event manipulation. For example, here we say some event is saved successfully. This may be confusing to the end user, because it's actually a custom event definition, not an event instance. We probably should use event definition through out this file.

Dashboard might have the same problem.